packages/core/src/registry/assert.ts

```diff
@@ -0,0 +1,44 @@
+import { FastifyRequest } from "fastify";
+import { RESTMethods } from "../loader/rest";
+
+export type AssertData = {
+  body: FastifyRequest["body"];
+  query: FastifyRequest["query"];
+  headers: FastifyRequest["headers"];
+};
+
+export default class Assert {
+  public request: Map<string, AssertData[]> = new Map();
```

Comment:
Not map but an array please. You might have 5 calls to an endpoint, you want to register all calls to for example assert on number of calls received

Reply:
Renamed to store.